docs/API.md

```diff
@@ -981,7 +992,8 @@ An object with the following properties:
 
 #### `"result"`
 
-* `"creation_time"`: A *timestamp*. The time at which the *test* was enqueued (in UTC).
+* `"creation_time"`: **Deprecated**. A *timestamp*. The time at which the *test* was enqueued (in UTC).
+* `"created_at"`: A *timestamp*. The time at which the *test* was enqueued (in UTC).
```

Comment:
The key says "created" but the documentation says "enqueued". I think it is unnecessary to have that confusion. Both could work, but "created" has been selected in most places.

If both has *timestamp*, is it just a matter of key name change? No, there is more to it.
```
* `"creation_time"`: **Deprecated**. A *timestamp* in deprecated representation. The time in UTC at which the *test* was created.
* `"created_at"`: A *timestamp* in prescribed representation. The time at in UTC which the *test* was created.
```

Or with my alternative when "timestamp" is defined:

```
* `"creation_time"`: **Deprecated**. A *deprecated timestamp*. The time in UTC at which the *test* was created.
* `"created_at"`: A *timestamp*. The time at in UTC which the *test* was created.
```

Reply:
> No, there is more to it.

Good point. I applied your suggested changes.